### ISSUE TITLE
Wallaby with vitest doesn't run all tests

### ISSUE THREAD

**Issue by sandervanheste1995** [wallabyjs/public#3033]
When using Wallaby Start with Vitest, it only runs a subset of all project tests.
When using 'Smart start', every test file that is opened runs correctly, but there seems to be no 
way to run all project tests automatically.

**Comment by smcenlly** [comment]
Could you please provide your Wallaby Diagnostics report after starting Wallaby with "Start" **not "Smart Start"?** 

Instructions for generating the report are in the [new issue template](https://github.com/wallabyjs/public/issues/new/choose).

**Comment by panick-carousel** [comment]
We are seeing this issue too.  Please see diagnostic report below.  Thank you for adding vitest to wallabyjs - really appreciate it 👍 

```
{
  editorVersion: '1.69.1',
  pluginVersion: '1.0.345',
  editorType: 'VSCode',
  osVersion: 'linux 5.17.13-300.fc36.x86_64',
  nodeVersion: 'v18.6.0',
  coreVersion: '1.0.1306',
  checksum: 'ODlmZDNhYTVkYzlkMzJhMzUyMzlmYjRlYzlhNjcwZDUsMTY2NjY1NjAwMDAwMCww',
  config: {
    diagnostics: {
      vitest: {
        file: {
          config: "import { defineConfig } from 'vite'\n" +
            "import react from '@vitejs/plugin-react'\n" +
            "import { visualizer } from 'rollup-plugin-visualizer'\n" +
            "import path from 'path'\n" +
            "import { createRequire } from 'module'\n" +
            'const require = createRequire(import.meta.url)\n' +
            '\n' +
            '// dependencies are split out here to test pure bundle size\n' +
            "const { dependencies } = require('./package.json')\n" +
            "dependencies['@noble/hashes/sha256'] = true\n" +
            "dependencies['multiformats/cid'] = true\n" +
            "dependencies['multiformats/block'] = true\n" +
            "dependencies['multiformats/codecs/raw'] = true\n" +
            "dependencies['multiformats/hashes/hasher'] = true\n" +
            "dependencies['@libp2p/crypto'] = true\n" +
            "dependencies['uint8arrays/to-string'] = true\n" +
            "dependencies['uint8arrays/from-string'] = true\n" +
            '\n' +
            '// https://vitejs.dev/config/\n' +
            'export default defineConfig({\n' +
            '  plugins: [react()],\n' +
            '  optimizeDeps: {\n' +
            '    // target: es2020 added as workaround to make big ints work\n' +
            '    // https://github.com/vitejs/vite/issues/9062#issuecomment-1182818044\n' +
            '    esbuildOptions: {\n' +
            "      target: 'es2020',\n" +
            '    },\n' +
            '  },\n' +
            '  build: {\n' +
            "    target: 'esnext',\n" +
            '    minify: true,\n' +
            '    rollupOptions: {\n' +
            "      plugins: [visualizer({ filename: './dist/vis.html' })],\n" +
            '      external: Object.keys(dependencies),\n' +
            '    },\n' +
            '    lib: {\n' +
            "      formats: ['es'],\n" +
            "      entry: path.resolve('./src/index.mjs'),\n" +
            '    },\n' +
            '  },\n' +
            '  define: {\n' +
            "    'process.env.NODE_DEBUG': 'false',\n" +
            '  },\n' +
            '  test: { globals: true },\n' +
            '})\n'
        },
        config: {
          allowOnly: true,
          watch: true,
          globals: true,
          environment: 'node',
          threads: true,
          clearMocks: false,
          restoreMocks: false,
          mockReset: false,
          include: [ '**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}' ],
          exclude: [ '**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**' ],
          testTimeout: 5000,
          hookTimeout: 10000,
          teardownTimeout: 1000,
          isolate: true,
          watchExclude: [ '**/node_modules/**', '**/dist/**' ],
          forceRerunTriggers: [ '**/package.json/**', '**/vitest.config.*/**', '**/vite.config.*/**' ],
          update: false,
          reporters: [ 'default' ],
          silent: false,
          ui: false,
          uiBase: '/__vitest__/',
          open: true,
          css: { include: [ {} ] },
          coverage: {
            enabled: false,
            clean: true,
            cleanOnRerun: false,
            reportsDirectory: '<homeDir>/repos/dreamcatcher-stack/pkg/interblock/coverage',
            excludeNodeModules: true,
            exclude: [
              'coverage/**',
              'dist/**',
              'packages/*/test{,s}/**',
              '**/*.d.ts',
              'cypress/**',
              'test{,s}/**',
              'test{,-*}.{js,cjs,mjs,ts,tsx,jsx}',
              '**/*{.,-}test.{js,cjs,mjs,ts,tsx,jsx}',
              '**/*{.,-}spec.{js,cjs,mjs,ts,tsx,jsx}',
              '**/__tests__/**',
              '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress}.config.{js,cjs,mjs,ts}',
              '**/.{eslint,mocha,prettier}rc.{js,cjs,yml}'
            ],
            reporter: [ 'text', 'html' ],
            allowExternal: false,
            extension: [
              '.js',  '.cjs',
              '.mjs', '.ts',
              '.tsx', '.jsx',
              '.vue', '.svelte'
            ],
            tempDirectory: '<homeDir>/repos/dreamcatcher-stack/pkg/interblock/coverage/tmp'
          },
          fakeTimers: { loopLimit: 10000, shouldClearNativeTimers: true, toFake: [ 'setTimeout', 'clearTimeout', 'setInterval', 'clearInterval', 'setImmediate', 'clearImmediate', 'Date' ] },
          maxConcurrency: 5,
          defines: {},
          root: '<homeDir>/repos/dreamcatcher-stack/pkg/interblock',
          deps: { inline: [ {}, {}, {}, {}, {}, {}, '@nuxt/test-utils' ] },
          snapshotOptions: { snapshotFormat: {}, updateSnapshot: 'new' },
          setupFiles: [],
          cache: { dir: '<homeDir>/repos/dreamcatcher-stack/pkg/interblock/node_modules/.vitest' },
          sequence: {},
          package: {
            version: '0.19.1',
            urls: { hooks: 'file://<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/runners/node/hooks.mjs' },
            paths: { root: '<homeDir>/repos/dreamcatcher-stack/node_modules/vitest', dist: '<homeDir>/repos/dreamcatcher-stack/node_modules/vitest/dist' }
          }
        }
      }
    },
    testFramework: { version: 'vitest@0.14.0', configurator: 'vitest@0.14.0', reporter: 'vitest@0.14.0', starter: 'vitest@0.14.0', autoDetected: true },
    preserveComments: false,
    extractComments: true,
    files: [
      { pattern: '**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}', ignore: true, trigger: true, load: true },
      { pattern: '**/node_modules/**', ignore: true, trigger: true, load: true },
      { pattern: '**/dist/**', ignore: true, trigger: true, load: true },
      { pattern: '**/cypress/**', ignore: true, trigger: true, load: true },
      { pattern: '**/.{idea,git,cache,output,temp}/**', ignore: true, trigger: true, load: true },
      { pattern: '**/*.*', ignore: false, trigger: true, load: true, order: 1 }
    ],
    tests: [
      { pattern: '**/node_modules/**', ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/dist/**', ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/cypress/**', ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/.{idea,git,cache,output,temp}/**', ignore: true, trigger: true, load: true, test: true },
      { pattern: '**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}', ignore: false, trigger: true, load: true, test: true, order: 2 }
    ],
    workers: { initial: 1, regular: 1, recycle: false },
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    logLimits: { inline: { depth: 5, elements: 5000 }, values: { default: { stringLength: 200 }, autoExpand: { elements: 5000, stringLength: 8192, depth: 10 } } },
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: {
      type: 'node',
      params: { runner: '--experimental-loader=file://<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/runners/node/hooks.mjs' },
      runner: '<homeDir>/.fnm/node-versions/v18.6.0/installation/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: undefined,
    symlinkNodeModules: true,
    configCode: 'auto.detect#1187933729'
  },
  packageJSON: {
    dependencies: {
      '@ipld/dag-cbor': '^7.0.2',
      '@ipld/dag-pb': '^2.1.17',
      '@noble/hashes': '^1.1.2',
      ajv: '^8.11.0',
      'ajv-formats': '^2.1.1',
      'ansi-colors-browserify': '^4.1.2',
      'assert-fast': '^1.0.1',
      callsites: '^4.0.0',
      columnify: '^1.6.0',
      debug: '^4.3.3',
      'fast-deep-equal': '^3.1.3',
      immutable: '^4.1.0',
      'ipfs-core': '^0.15.4',
      'ipld-hashmap': '^2.1.11',
      'ipld-schema': '^2.0.2',
      'pad-left': '^2.1.0',
      'path-browserify': '^1.0.1',
      'pretty-bytes': '^6.0.0',
      'serialize-error': '^11.0.0',
      uint8arrays: '^3.0.0'
    },
    devDependencies: {
      '@types/react': '^18.0.15',
      '@types/react-dom': '^18.0.6',
      '@vitejs/plugin-react': '^2.0.0',
      '@vitest/ui': '^0.19.1',
      benchmark: '^2.1.4',
      chai: '^4.3.6',
      eslint: '^8.20.0',
      'eslint-config-prettier': '^8.5.0',
      'eslint-plugin-prettier': '^4.2.1',
      'eslint-plugin-react': '^7.30.1',
      prettier: '^2.7.1',
      react: '^18.2.0',
      'react-dom': '^18.2.0',
      'rollup-plugin-visualizer': '^5.7.1',
      'supports-color': '^9.2.2',
      'trace-unhandled': '^2.0.1',
      vite: '^3.0.3',
      vitest: '^0.19.1'
    }
  },
  fs: { numberOfFiles: 175 },
  debug: [
    '2022-07-29T00:00:08.109Z angular/cli config Angular CLI not found.\n',
    "2022-07-29T00:00:08.111Z jest/config Error: Module jest-cli is not found in '<homeDir>/repos/dreamcatcher-stack/pkg/interblock'.\n" +
      '    at Object.<anonymous> (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:31:9440)\n' +
      '    at r (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:30:24761)\n' +
      '    at Object.next (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:30:24056)\n' +
      '    at <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:30:23965\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at n (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:30:23759)\n' +
      '    at Object.loadJest (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:31:7038)\n' +
      '    at <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:15529\n' +
      '    at r (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:20269)\n' +
      '    at Object.next (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19564)\n' +
      '    at <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19473\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19267)\n' +
      '    at r (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:15217)\n' +
      '    at <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:18944\n' +
      '    at r (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:20269)\n' +
      '    at Object.next (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19564)\n' +
      '    at <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19473\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:19267)\n' +
      '    at n (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:19:18825)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:15:20165)\n' +
      '    at r (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:15:12061)\n' +
      '    at Object.next (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:15:11356)\n' +
      '    at o (<homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/wallaby592669/server.js:15:11105)\n',
    '2022-07-29T00:00:08.113Z vitest/config Detected Vitest.\n',
    '2022-07-29T00:00:08.580Z project Wallaby Node version: v18.6.0\n',
    '2022-07-29T00:00:08.580Z project Wallaby config: <homeDir>/repos/dreamcatcher-stack/pkg/interblock/auto.detect\n',
    '2022-07-29T00:00:08.788Z project File cache: <homeDir>/.vscode-server/extensions/wallabyjs.wallaby-vscode-1.0.345/projects/7aca19534f3cfd57\n',
    '2022-07-29T00:00:08.829Z uiService Listening port 51235\n',
    '2022-07-29T00:00:08.860Z workers Parallelism for initial run: 1, for regular run: 1\n',
    '2022-07-29T00:00:08.860Z workers Starting run worker instance #0\n',
    '2022-07-29T00:00:08.861Z workers Web server is listening at 36499\n',
    '2022-07-29T00:00:08.868Z project Stopping process pool\n',
    '2022-07-29T00:00:08.868Z project File cache is up-to-date, starting full test run\n',
    '2022-07-29T00:00:08.869Z project Test run started; run priority: 3\n',
    '2022-07-29T00:00:08.870Z project Running all tests\n',
    '2022-07-29T00:00:08.873Z workers Starting test run, priority: 3\n',
    '2022-07-29T00:00:08.873Z nodeRunner Starting sandbox [worker #0, session #i8s0h]\n',
    '2022-07-29T00:00:08.873Z nodeRunner Preparing sandbox [worker #0, session #i8s0h]\n',
    '2022-07-29T00:00:08.990Z workers Started run worker instance (delayed) #0\n',
    '2022-07-29T00:00:08.991Z nodeRunner Prepared sandbox [worker #0, session #i8s0h]\n',
    '2022-07-29T00:00:08.991Z workers [worker #0, session #i8s0h] Running tests in sandbox\n',
    '2022-07-29T00:00:09.238Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-07-29T00:00:09.336Z fs File deleted: vite.config.js.timestamp-1659052809153.mjs\n',
    '2022-07-29T00:00:09.338Z project Test run is not cancelled because cancel requester does not have enough priority to cancel the run\n',
    '2022-07-29T00:00:09.362Z workers Scheduling Vitest Run (i8s0h): 2022-07-29T00:00:09.309Z\n',
    '2022-07-29T00:00:11.714Z workers [i8s0h.15] Test executed: recursive crush\n',
    '2022-07-29T00:00:11.727Z workers [i8s0h.15] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:11.739Z workers [i8s0h.15] Test executed: basic\n',
    '2022-07-29T00:00:11.742Z workers [i8s0h.15] Test executed: multiple keys\n',
    '2022-07-29T00:00:11.743Z workers [i8s0h.15] Test executed: with class\n',
    '2022-07-29T00:00:11.748Z workers [i8s0h.15] Test executed: throws on existing key\n',
    '2022-07-29T00:00:12.344Z workers [i8s0h.12] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:12.499Z workers [i8s0h.14] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:12.500Z workers [i8s0h.12] Test executed: locks\n',
    '2022-07-29T00:00:12.501Z workers [i8s0h.14] Test executed: no params makes unknown address\n',
    '2022-07-29T00:00:12.502Z workers [i8s0h.14] Test executed: generate from Pulse\n',
    '2022-07-29T00:00:12.543Z workers [i8s0h.12] Test executed: multiple locks\n',
    '2022-07-29T00:00:12.559Z workers [i8s0h.9] Test executed: loopback transmissions\n',
    '2022-07-29T00:00:12.568Z workers [i8s0h.9] Test executed: loopback promises\n',
    '2022-07-29T00:00:12.576Z workers [i8s0h.9] Test executed: receive interpulse\n',
    '2022-07-29T00:00:12.586Z workers [i8s0h.9] Test executed: remove transmission\n',
    '2022-07-29T00:00:12.593Z workers [i8s0h.9] Test executed: softpulse for pooling\n',
    '2022-07-29T00:00:12.597Z workers [i8s0h.9] Test executed: genesis pulse\n',
    '2022-07-29T00:00:12.600Z workers [i8s0h.10] Test executed: channel cannot have address changed once resolved\n',
    '2022-07-29T00:00:12.601Z workers [i8s0h.9] Test executed: pierce pulse\n',
    '2022-07-29T00:00:12.601Z workers [i8s0h.10] Test executed: rxReply always selected before rxRequest\n',
    '2022-07-29T00:00:12.604Z workers [i8s0h.9] Test executed: pending\n',
    '2022-07-29T00:00:12.606Z workers [i8s0h.10] Test executed: rxReply( request ) throws if non existant channel in request\n',
    '2022-07-29T00:00:12.611Z workers [i8s0h.10] Test executed: empty string cannot be used as channel name\n',
    '2022-07-29T00:00:12.613Z workers [i8s0h.10] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:12.616Z workers [i8s0h.9] Test executed: public connection\n',
    '2022-07-29T00:00:12.637Z workers [i8s0h.10] Test executed: creates default\n',
    '2022-07-29T00:00:12.639Z workers [i8s0h.9] Test executed: precedent multiple pulses ahead\n',
    '2022-07-29T00:00:12.640Z workers [i8s0h.9] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:12.641Z workers [i8s0h.10] Test executed: parent is unknown by default\n',
    '2022-07-29T00:00:12.645Z workers [i8s0h.9] Test executed: basic\n',
    '2022-07-29T00:00:12.646Z workers [i8s0h.10] Test executed: can only set Channel instances\n',
    '2022-07-29T00:00:12.652Z workers [i8s0h.9] Test executed: birth child\n',
    '2022-07-29T00:00:12.653Z workers [i8s0h.10] Test executed: transmit to downlink\n',
    '2022-07-29T00:00:12.659Z workers [i8s0h.10] Test executed: open uplink\n',
    '2022-07-29T00:00:12.663Z workers [i8s0h.10] Test executed: large network\n',
    '2022-07-29T00:00:13.021Z workers [i8s0h.4] Test executed: nested calls to wrapReduce\n',
    '2022-07-29T00:00:13.021Z workers [i8s0h.4] Test executed: duplicate requests permitted in different calls\n',
    '2022-07-29T00:00:13.022Z workers [i8s0h.4] Test executed: throw on incomplete accumulator\n',
    '2022-07-29T00:00:13.022Z workers [i8s0h.4] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.023Z workers [i8s0h.4] Test executed: nested hooks awaited\n',
    '2022-07-29T00:00:13.023Z workers [i8s0h.4] Test executed: void return\n',
    '2022-07-29T00:00:13.024Z workers [i8s0h.4] Test executed: plain reply\n',
    '2022-07-29T00:00:13.024Z workers [i8s0h.4] Test executed: nested parallel hooks do not collide\n',
    '2022-07-29T00:00:13.025Z workers [i8s0h.4] Test executed: reduction must be an object\n',
    '2022-07-29T00:00:13.025Z workers [i8s0h.4] Test executed: duplicate requests permitted in same call\n',
    '2022-07-29T00:00:13.026Z workers [i8s0h.4] Test executed: resolved result\n',
    '2022-07-29T00:00:13.027Z workers [i8s0h.4] Test executed: reject\n',
    '2022-07-29T00:00:13.129Z workers [i8s0h.4] Test executed: timeout exceeded\n',
    '2022-07-29T00:00:13.150Z workers [i8s0h.4] Test executed: single await\n',
    '2022-07-29T00:00:13.163Z workers [i8s0h.4] Test executed: rejection\n',
    '2022-07-29T00:00:13.230Z workers [i8s0h.7] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.351Z workers [i8s0h.13] Test executed: reject for unknown chainId\n',
    '2022-07-29T00:00:13.352Z workers [i8s0h.13] Test executed: reject for unpierced chain\n',
    '2022-07-29T00:00:13.353Z workers [i8s0h.13] Test executed: duplicate piercings only included once\n',
    '2022-07-29T00:00:13.353Z workers [i8s0h.13] Test executed: piercings already in current block excluded\n',
    '2022-07-29T00:00:13.353Z workers [i8s0h.13] Test executed: acl blocked system action is rejected\n',
    '2022-07-29T00:00:13.354Z workers [i8s0h.13] Test executed: unpierced chain rejects attempted outbound pierce\n',
    '2022-07-29T00:00:13.355Z workers [i8s0h.13] Test executed: reject attempt to make channel named .@@io even if pierced\n',
    '2022-07-29T00:00:13.355Z workers [i8s0h.13] Test executed: unpierce during pierce execution drops all other pierces\n',
    '2022-07-29T00:00:13.356Z workers [i8s0h.13] Test executed: opening pierce channel alone does not cause extra lineage\n',
    '2022-07-29T00:00:13.357Z workers [i8s0h.13] Test executed: always at least one tx in the channel to keep count\n',
    '2022-07-29T00:00:13.357Z workers [i8s0h.13] Test executed: change to unpierced removes .@@io channel\n',
    '2022-07-29T00:00:13.358Z workers [i8s0h.13] Test executed: if not in block piercing is not purged from db\n',
    '2022-07-29T00:00:13.358Z workers [i8s0h.13] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.359Z workers [i8s0h.13] Test executed: do not txInterblocks to .@@io channel\n',
    '2022-07-29T00:00:13.359Z workers [i8s0h.13] Test executed: multiple simultaneous pings maintain order\n',
    '2022-07-29T00:00:13.435Z workers [i8s0h.11] Test executed: throw if pending and tx a request to self\n',
    '2022-07-29T00:00:13.437Z workers [i8s0h.11] Test executed: wait for all promises\n',
    '2022-07-29T00:00:13.438Z workers [i8s0h.11] Test executed: awaiting on self alone rejects\n',
    '2022-07-29T00:00:13.439Z workers [i8s0h.11] Test executed: covenant must make requests in same order\n',
    '2022-07-29T00:00:13.440Z workers [i8s0h.11] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.441Z workers [i8s0h.11] Test executed: loopback cleared immediately\n',
    '2022-07-29T00:00:13.465Z workers [i8s0h.8] Test executed: nothing to transmit avoids making a block\n',
    '2022-07-29T00:00:13.465Z workers [i8s0h.8] Test executed: config changes cause new block\n',
    '2022-07-29T00:00:13.465Z workers [i8s0h.8] Test executed: rename alias does not cause interblock\n',
    '2022-07-29T00:00:13.466Z workers [i8s0h.8] Test executed: automatic promises\n',
    '2022-07-29T00:00:13.466Z workers [i8s0h.8] Test executed: tick with no response is an instant resolve\n',
    '2022-07-29T00:00:13.466Z workers [i8s0h.8] Test executed: resolving an alias causes lineage fork\n',
    '2022-07-29T00:00:13.466Z workers [i8s0h.8] Test executed: reject if piercings for unpierced reducer\n',
    '2022-07-29T00:00:13.467Z workers [i8s0h.8] Test executed: triangular test for large numbers of pooled lineage\n',
    '2022-07-29T00:00:13.467Z workers [i8s0h.8] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.467Z workers [i8s0h.8] Test executed: double digit blocks\n',
    '2022-07-29T00:00:13.500Z workers [i8s0h.7] Test executed: @@INIT\n',
    '2022-07-29T00:00:13.500Z workers [i8s0h.7] Test executed: throw on @@INIT bubbles up\n',
    '2022-07-29T00:00:13.768Z workers [i8s0h.5] Test executed: no double open\n',
    '2022-07-29T00:00:13.768Z workers [i8s0h.5] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:13.769Z workers [i8s0h.5] Test executed: basic\n',
    '2022-07-29T00:00:13.770Z workers [i8s0h.5] Test executed: rejects on unopenable path\n',
    '2022-07-29T00:00:14.057Z workers [i8s0h.6] Test executed: spawn uses hash for seeding inside action\n',
    '2022-07-29T00:00:14.060Z workers [i8s0h.6] Test executed: cannot be more than one outstanding @@OPEN request in a channel\n',
    '2022-07-29T00:00:14.061Z workers [i8s0h.6] Test executed: missing parents with open children skips straight to children\n',
    '2022-07-29T00:00:14.061Z workers [i8s0h.6] Test executed: 3 levels deep\n',
    '2022-07-29T00:00:14.061Z workers [i8s0h.6] Test executed: random path opening and closing and reopening\n',
    '2022-07-29T00:00:14.062Z workers [i8s0h.6] Test executed: simultaneous requests with common parent\n',
    '2022-07-29T00:00:14.062Z workers [i8s0h.6] Test executed: multiple requests from same chain result in single uplink\n',
    '2022-07-29T00:00:14.062Z workers [i8s0h.6] Test executed: connect on existing is the same as move\n',
    '2022-07-29T00:00:14.063Z workers [i8s0h.6] Test executed: connect resolves an address without purging queued actions\n',
    '2022-07-29T00:00:14.063Z workers [i8s0h.6] Test executed: connect on existing unknown transmits all queued actions\n',
    '2022-07-29T00:00:14.064Z workers [i8s0h.6] Test executed: connect on operational channel throws\n',
    '2022-07-29T00:00:14.067Z workers [i8s0h.6] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:14.067Z workers [i8s0h.6] Test executed: spawn is implicitly awaited\n',
    '2022-07-29T00:00:14.068Z workers [i8s0h.6] Test executed: error if no spawn await\n',
    '2022-07-29T00:00:14.068Z workers [i8s0h.6] Test executed: system ping\n',
    '2022-07-29T00:00:14.574Z workers [i8s0h.3] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:14.583Z workers [i8s0h.3] Test executed: ping single\n',
    '2022-07-29T00:00:14.583Z workers [i8s0h.3] Test executed: ping many times\n',
    '2022-07-29T00:00:14.584Z workers [i8s0h.3] Test executed: create child\n',
    '2022-07-29T00:00:14.585Z workers [i8s0h.3] Test executed: spawn many times\n',
    '2022-07-29T00:00:14.797Z workers [i8s0h.2] Test executed: opens up a path\n',
    '2022-07-29T00:00:14.797Z workers [i8s0h.2] Test executed: coordinates with simultaneous path openings\n',
    '2022-07-29T00:00:14.798Z workers [i8s0h.2] Test executed: detects changes in filesystem\n',
    '2022-07-29T00:00:14.798Z workers [i8s0h.2] Test executed: cd rejects if non existent path\n',
    '2022-07-29T00:00:14.798Z workers [i8s0h.2] Test executed: absolute path\n',
    '2022-07-29T00:00:14.799Z workers [i8s0h.2] Test executed: parent path\n',
    '2022-07-29T00:00:14.799Z workers [i8s0h.2] Test executed: invalid parent path rejects\n',
    '2022-07-29T00:00:14.799Z workers [i8s0h.2] Test executed: grandchild can spawn\n',
    '2022-07-29T00:00:14.800Z workers [i8s0h.2] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:14.800Z workers [i8s0h.2] Test executed: parallel request is processed\n',
    '2022-07-29T00:00:14.800Z workers [i8s0h.2] Test executed: cd to valid nested path\n',
    '2022-07-29T00:00:14.800Z workers [i8s0h.2] Test executed: cd errors on garbage path\n',
    '2022-07-29T00:00:14.801Z workers [i8s0h.2] Test executed: cd errors on nested garbage path\n',
    '2022-07-29T00:00:14.801Z workers [i8s0h.2] Test executed: . is resolved\n',
    '2022-07-29T00:00:14.801Z workers [i8s0h.2] Test executed: .. is valid\n',
    '2022-07-29T00:00:14.801Z workers [i8s0h.2] Test executed: cd .. at root stays at root\n',
    '2022-07-29T00:00:14.802Z workers [i8s0h.2] Test executed: list current directory\n',
    '2022-07-29T00:00:14.802Z workers [i8s0h.2] Test executed: list remote directory\n',
    '2022-07-29T00:00:14.802Z workers [i8s0h.2] Test executed: throws on invalid directory\n',
    '2022-07-29T00:00:14.802Z workers [i8s0h.2] Test executed: throws on invalid nested directory\n',
    '2022-07-29T00:00:14.803Z workers [i8s0h.2] Test executed: throws on invalid double nested directory\n',
    '2022-07-29T00:00:14.803Z workers [i8s0h.2] Test executed: throws on shallow invalid nested directory\n',
    '2022-07-29T00:00:14.803Z workers [i8s0h.2] Test executed: throws on deep invalid nested directory\n',
    '2022-07-29T00:00:14.803Z workers [i8s0h.2] Test executed: root path when cd is child\n',
    '2022-07-29T00:00:14.803Z workers [i8s0h.2] Test executed: basic\n',
    '2022-07-29T00:00:14.804Z workers [i8s0h.2] Test executed: normalize tests\n',
    '2022-07-29T00:00:14.804Z workers [i8s0h.2] Test executed: deep child runs custom covenant\n',
    '2022-07-29T00:00:15.076Z workers [i8s0h.1] Test executed: can only add customer if provide valid data\n',
    '2022-07-29T00:00:15.076Z workers [i8s0h.1] Test executed: add customer with test data using .processes/addTestCustomer\n',
    '2022-07-29T00:00:15.077Z workers [i8s0h.1] Test executed: imports customer data\n',
    '2022-07-29T00:00:15.077Z workers [i8s0h.1] Test executed: bulk data upload\n',
    '2022-07-29T00:00:15.078Z workers [i8s0h.1] Test executed: update customer name\n',
    '2022-07-29T00:00:15.078Z workers [i8s0h.1] Test executed: add a service to a customer\n',
    '2022-07-29T00:00:15.079Z workers [i8s0h.1] Test executed: cannot alter the structure of the application in any way\n',
    '2022-07-29T00:00:15.079Z workers [i8s0h.1] Test executed: 20,000 test customers added\n',
    '2022-07-29T00:00:15.080Z workers [i8s0h.1] Loaded unknown number of test(s)\n',
    '2022-07-29T00:00:15.080Z workers [i8s0h.1] Test executed: deploys app\n',
    '2022-07-29T00:00:15.081Z workers [i8s0h.1] Test executed: list customers basic\n',
    '2022-07-29T00:00:15.107Z workers Vitest Run Complete (i8s0h): 2022-07-29T00:00:15.098Z\n',
    '2022-07-29T00:00:15.117Z workers [i8s0h] Run 132 test(s), skipped 0 test(s)\n',
    '2022-07-29T00:00:15.118Z workers [i8s0h] Sandbox is responsive, closing it\n',
    '2022-07-29T00:00:15.147Z project Test run finished\n',
    '2022-07-29T00:00:15.147Z project Processed console.log entries\n',
    '2022-07-29T00:00:15.148Z project Processed loading sequences\n',
    '2022-07-29T00:00:15.149Z project Processed executed tests\n',
    '2022-07-29T00:00:15.168Z project Processed code coverage\n',
    '2022-07-29T00:00:15.244Z project Test run result processed and sent to IDE\n',
    '2022-07-29T00:00:15.245Z project Changes did not trigger any tests\n'
  ]
}
```

**Comment by panick-carousel** [comment]
Only runs 62 tests, but `npx vitest` runs 93 tests 🤷 

![image](https://user-images.githubusercontent.com/84829404/181656662-b077d234-0c1d-460b-9c28-d18230415dce.png)

**Comment by panick-carousel** [comment]
running in a lerna monorepo what you can see all our config here: https://github.com/dreamcatcher-tech/dreamcatcher-stack

**Comment by smcenlly** [comment]
Thanks for the sample repo and your diagnostics report. It allowed us to reproduce the error.

This has been fixed in Wallaby core, `v1.0.1307`.

**Comment by panick-carousel** [comment]
thank you - we confirm this fix
